provider/src/main/java/com/raygun/raygun4android/rum/RUM.java

```diff
@@ -331,9 +242,10 @@ public static void sendRUMTimingEvent(
                     ? new RaygunUserInfo(null, null, null, null)
                     : RaygunClient.getUser();
 
+            // TODO(Miguel Beltran): type = p, what does it mean?
             RaygunRUMTimingMessage timingMessage =
                 new RaygunRUMTimingMessage.Builder(
-                    eventType == RaygunRUMEventType.ACTIVITY_LOADED ? "p" : "n")
+                    eventType == RaygunRUMEventType.ACTIVITY_LOADED || eventType == RaygunRUMEventType.FRAGMENT_LOADED ? "p" : "n")
                     .duration(milliseconds)
                     .build();
```

Comment:
From my understanding, `p` is used for any screen view, so we should use `p` for both loading Activities and Fragments.